nix-darwin/roles/buildkite-agent.nix

```diff
@@ -8,6 +8,6 @@
   ];
 
   services = {
-    dd-agent.tags = [" group:buildkite-agents"];
+    dd-agent.tags = [" group:buildkite-agents" "group:macos"];
```

Comment:
we should be able to remove the redundant space before group, that was to fix a buildkite module bug